### ISSUE TITLE
Print Authors Multiple Pages

### ISSUE THREAD

**Issue by limeb** [thomaspark/pubcss#14]
For some reason the authors are spread across two pages when printing in Chrome in this example.  

https://thomaspark.co/project/pubcss/demo/acm-sig-sample-web-theme.html

Maybe this is the intended behavior but I am not really sure.  Also the abstract appears rather far down the page for some reason.

**Comment by limeb** [comment]
@thomaspark  Yes Chrome.  So how do I generate a reliable pdf if not using Chrome's print command?

**Comment by thomaspark** [comment]
You can use Prince on the command line to generate a PDF from your HTML. It handles a lot of the new CSS features better than the browsers.

https://github.com/thomaspark/pubcss#quick-start
https://www.princexml.com/

**Comment by limeb** [comment]
hmm I might invest some time into this.  I really really dislike the idea of using a 3rd party software that isn't well incredibly popular for this.  I mean prince must be using webkit(I'm guessing) or something right?  Which means the CSS is somewhere in there.

**Comment by thomaspark** [comment]
Understandable, check out #12 for work towards using Puppeteer instead.

**Comment by limeb** [comment]
So because HTML doesn't support footnotes natively I am throwing html out to probably use markdown or latex.  HTML has no agreed upon syntax unfortunately.
